src/formio/components/Radio.stories.mdx

```diff
@@ -43,7 +43,7 @@ export const Template = ({ id = '', label = '', ...args}) => (
           "value": "radioB"
       },
       {
-          "label": "Optie C with a <a href=\"https://github.com/open-formulieren/\" target=\"_blank\">link</a>",
```

Comment:
We said the other day that we don't want to support HTML in components, so I thought it would be good to remove this example